contracts/protocol/clients/voucher/BosonVoucher.sol

```diff
@@ -707,19 +707,19 @@ contract BosonVoucher is IBosonVoucher, BeaconClientBase, OwnableUpgradeable, ER
                     if (start > _tokenId) {
                         // Split low and search again if target too high
                         high = mid;
-                    } else if (start + range.minted - 1 >= _tokenId) {
-                        // Is token in target's minted range?
+                    } else if (start + range.length - 1 >= _tokenId) {
+                        // Is token in target's reserved range?
+
+                        if (start + range.minted - 1 >= _tokenId) {
+                            // Has it been pre-minted?
 
-                        // It is committable if it has not been burned
-                        if (_tokenId > range.lastBurnedTokenId) {
-                            committable = true;
-                            offerId = currentOfferId;
+                            // It is committable if it has not been burned
+                            if (_tokenId > range.lastBurnedTokenId) {
```

Comment:
could we not merge the two if statements together ?

```if ((start + range.minted - 1 >= _tokenId) && (_tokenId > range.lastBurnedTokenId)) { ```

Does that make it more readable ?

Reply:
I merged them, makes sense.